platform-hub-api/app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ProjectsController < ApiJsonController
+
+  before_action :find_project, only: [ :show, :update, :destroy, :memberships, :add_membership, :remove_membership ]
+  before_action :find_user, only: [ :add_membership, :remove_membership ]
+
+  skip_authorization_check :only => [ :index, :show, :memberships ]
+  authorize_resource except: [ :index, :show, :memberships ]
+
+  # GET /projects
+  def index
+    @projects = Project.order(:shortname)
+
+    render json: @projects
+  end
+
+  # GET /projects/:id
+  def show
+    render json: @project
+  end
+
+  # POST /projects
+  def create
+    @project = Project.new(project_params)
+
+    if @project.save
```

Comment:
Not in this particular case? Note, there's an `else` branch that ensures we respond with a `422 Unprocessable Entity` response (and show what errors caused the save to fail).